evaluate.js

```diff
@@ -0,0 +1,85 @@
+Array.prototype.clean = function() {
+    for(let i = 0; i < this.length; i++) {
+        if(this[i] === "") {
+            this.splice(i, 1);
+        }
+    }
+    return this;
+}
+
+function expressionToRpn(expression) {
```

Comment:
split it to smaller logical blocks
it would be better, and more readable

Reply:
slightly cleaned the function